### ISSUE TITLE
Choose descriptive css class names

### ISSUE THREAD

**Issue by cjessett** [brett099/prj-rev-bwfs-dasmoto#1]
(Subtlety) [Here](https://github.com/brett099/prj-rev-bwfs-dasmoto/blob/4a6ef4e37544354c12e2175b42b73563d5d18958/arts%26crafts/resources/css/style.css#L9) you should just use a single css class. Use a name that is descriptive of that section on the page.

**Comment by brett099** [comment]
Yeah that makes sence i just figured it might be easier to just use the classes. But you are right now that i think about it.
